pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -502,7 +507,8 @@ func (this *SnapshotManager) deleteSnapshot(peID astrolabe.ProtectedEntityID, ba
 
 	log.Infof("Step 2: Deleting the durable snapshot from s3")
 	if backupRepositoryName != "" && backupRepositoryName != constants.WithoutBackupRepository {
-		backupRepositoryCR, err := pluginClient.BackupdriverV1alpha1().BackupRepositories().Get(context.TODO(), backupRepositoryName, metav1.GetOptions{})
+		var backupRepositoryCR *backupdriverv1.BackupRepository
```

Comment:
Is it necessary to add this?

Reply:
While triaging the veritas issue noticed the error was being shadowed, noticed it from logs, hence decided to make this change.